### PR TITLE
feat: adding spirals to dapp-list

### PR DESCRIPTION
**Pull request by clague17**
> To make a submission to the Dapps Page, please complete the task lists
> below; otherwise remove them.

Your Dapp:

- [x] connects to Valora via WalletConnect V1 or DAppKit;
- [x] must be mobile-first or mobile responsive;
- [x] is in one of the existing categories; and
- [x] includes a link to user documentation.

### CODE REVIEW

**Comment by jeanregisser** [comment]
Thanks for the submission @clague17 🎉 

The logo is [not the right size](https://github.com/valora-inc/dapp-list/actions/runs/3047441308/jobs/4919289281), it needs to be a 256x256 png file.

Could you update it?

**Comment by clague17** [comment]
> Thanks for the submission @clague17 🎉
> 
> The logo is [not the right size](https://github.com/valora-inc/dapp-list/actions/runs/3047441308/jobs/4919289281), it needs to be a 256x256 png file.
> 
> Could you update it?

Updated! @jeanregisser

**Comment by gsalaiz-Valora** [comment]
Hi @clague17 is there a way to contact you? or please send us an email. partnerships@valoraapp.com

File: src/valora-dapp-list.json [hunk 1]
```diff
@@ -239,6 +239,12 @@
       "id": "pixpress",
       "categoryId": "exchanges",
       "url": "https://pixpress.pixelava.space"
+    },
+    {
+      "name": "Spirals",
+      "id": "spirals",
+      "categoryId": "social-impact",
+      "url": "https://demo.spirals.so"
```
Comment (helenamerk) [inline src/valora-dapp-list.json:247]: Can we update this to `https://app.spirals.so`

**Comment by MuckT** [comment]
@jeanregisser I've been able to steak and unsteak cUSD. This is looking good to me!

**Comment by clague17** [comment]
Hey @MuckT @jeanregisser would love to address any feedback to get this merged!

**Comment by MuckT** [comment]
Works great, the only feedback I have is the app name is listed in the Dapp List as "Spirals"; however, the Dapp name given in the prompt is "Climate Impact Staking". This might confuse users as they well be attempting to connect to "Spirals" via the Dapp list and see a prompt for "Climate Impact Staking".

**Review by MuckT — APPROVED** [review]
Excited to have this included in the dapp-list! 🚀

**Pull request merged**